infra/helm-charts/mongo/charts/ingestbackup/compose-backup.yml

```diff
@@ -6,7 +6,7 @@ services:
     build:
       dockerfile: Dockerfile-backup
       context: .
-    image: quay.io/humancellatlas/ingest-backup
+    image: quay.io/ebi-ait/ingest-kube-deployment:ingest-backup_20210929.1
```

Comment:
Why are we using date based tags rather than something more robust?

Reply:
@amnonkhen What do you mean by robust? What is wrong with date based tags?